### DISCUSSION TITLE
Throwing an Error exception

### DISCUSSION THREAD

**Discussion by kironia** [kironia/99balloons#4] [Q&A] [answered]
What kind of exception is the Error exception? I'm also curious about inheriting it.

**Comment by Stagyrite** [comment, marked as answer]
First, Error doesn't inherit from the RuntimeException class. Therefore, one can't call it a runtime exception. Nevertheless, it can be called an unchecked exception. Of course, it's not a checked exception.
As for inheriting it, one can create another Error. It's meant to be for unrecoverable errors like a thread death. One can't possibly bring a thread back to life.
Java 1.7 brought yet another constructor for this particular class. One can disable suppressing exceptions and thus any effect of the 'addSuppressed()' method. One can also disable filling in the execution stack trace, which means an error can come without a stack trace written in it.
